various copy changes

orishim opened the pull request:
closes #198

krisbitney commented:
I have a couple of questions:
What does "We dogfood your products" mean?
Roberto requested that his location be listed as Maracaibo, Venezuela. Is it alright to keep that?

orishim commented:
I changed it back to maracaibo

dogfooding:
https://en.wikipedia.org/wiki/Eating_your_own_dog_food

I changed the wording of the sentence to make it a bit clearer

krisbitney commented:
> I changed it back to maracaibo
> 
> dogfooding:
> https://en.wikipedia.org/wiki/Eating_your_own_dog_food
> 
> I changed the wording of the sentence to make it a bit clearer

Very cool
